src/triangulation.jl

```diff
@@ -34,7 +43,7 @@ function triangulation_indices(p::Polyhedron)
     Δs = typeof(Δ)[]
     incident_idx = Dict(h => Set(incidentpointindices(p, h)) for h in h_idx)
     is_weak_adjacent = Dict((hi, hj) => !isempty(incident_idx[hi] ∩ incident_idx[hj]) for hi in h_idx for hj in h_idx)
-    _triangulation(Δs, Δ, v_idx, h_idx, incident_idx, is_weak_adjacent)
+    _triangulation(Δs, Δ, v_idx, h_idx, incident_idx, is_weak_adjacent, fulldim(p))
```

Comment:
We might want to add this as optional argument to `triangulation_indices` in case someone wants to obtain triangulation of lower dimensional polyhedra

Reply:
Would it be ok to defer that to a follow-up PR?  That way this PR can be focused on just the bug fix, and the follow-up can make that extension and add the corresponding unit tests.